packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -360,47 +360,41 @@ class _GoogleMapState extends State<GoogleMap> {
       return;
     }
     final GoogleMapController controller = await _controller.future;
-    // ignore: unawaited_futures
-    controller._updateMapConfiguration(updates);
+    await controller._updateMapConfiguration(updates);
```

Comment:
In practice, none of these enclosing `_update*` methods are awaited, so this shouldn't change any behavior. Having clients of the wrapper method decide whether or not to await seemed like the right place for that decision, which is why I made this `await`, but I may be missing something.

Reply:
Returning from the (not-so-distant) future to point out that this is not correct; it changes the timing of updating the ivars, regardless of what the caller does. I'll be undoing these changes.